sources/packages/web/src/views/student/StudentDashboard.vue

```diff
@@ -1,33 +1,37 @@
 <template>
   <student-page-container layout-template="centered">
-    <formio
-      formName="studentwelcomepage"
+    <formio-container
+      formName="studentWelcomePage"
+      :formData="formData"
       @customEvent="goToStudentApplication"
-    ></formio>
+    />
   </student-page-container>
 </template>
 <script lang="ts">
-import { StudentRoutesConst } from "../../constants/routes/RouteConstants";
-import { computed } from "vue";
-import { useStore } from "vuex";
+import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
 import { useRouter } from "vue-router";
+import { onMounted, reactive } from "vue";
+import { StudentService } from "@/services/StudentService";
 
 export default {
   setup() {
-    const store = useStore();
     const router = useRouter();
-    const user = computed(() => store.state.student.profile);
-
+    const formData = reactive({ firstName: "" });
```

Comment:
I am not sure it this must e reactive. Did you check?

Reply:
as discussed passing the whole API response

Reply:
@ann-aot there is another comment from @dheepak-aot about getting the data from the store. I just replied to that also: https://github.com/bcgov/SIMS/pull/1433/files#r997474468